pages/glossary/clearly-labeled-location.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Clearly labeled location
+key: clearly-labeled-location
+unambiguous: true
+objective: false
+---
+
+Secondary information and alternative controls of functionality are often not displayed together with primary information or functionality. For example, an option to change a web page to dark mode may be placed on an options page instead of being available on every page and page state of a website. Another example is a maps application, where, instead of using GPS, an option is available in a dropdown menu to set the current location of the device. Such content should be placed in a clearly labeled location.
+
+The location of a target is said to be _clearly labeled_ when the target can be found by activating [instruments][] whose "description" either leads the user to find the target, or to another [page][web page] or page state from which this action can be repeated until the target is found.
+
+Whether or not the content is "clearly labeled" depends on the starting point of the search. If page A has a link with a clear "label" to some piece of content, then the location of the content is clearly labeled. Page B, which can be in the same website, may not have such a link or may have a link with an ambiguous link text, and so the location of the content starting from page B is not clearly labeled.
+
+For the purpose of the definition of "Clearly labeled location", a "description" is any text or other content with a [text alternative][], that allows any user to identify an element with a [semantic role][] that inherits from `widget`.
```

Comment:
@Jym77 I'm afraid I don't agree with you on this. We're defining "clearly labeled location" not "clearly labeled purpose". Therefore, your example of identify is spot on. 

If you look at the expectation it states the instrument has to be on the same page (or in a clearly labeled location), but it says nothing about how the purpose is conveyed to the user/evaluator. This rule just checks that the instrument exists (on the same web page or if it is in another page, in a location that is conveyed to the user/evaluator). Which is aligned with the SC that just requires that "there is a mechanism ..."

Reply:
I'm on board with this interpretation 😄 

But then, I'm challenging the word "description" which is way too close to "descriptive" used in other rules to mean "describe purpose" (and not "describe location").
I'm also become a bit uneasy with "clearly **labelled**" being something about a "description" (I know I've asked it to not be a "label"…) especially since WCAG requires "labels" to be "descriptive" which further the confusion…

What about "clearly identified location" and "identification" instead?
That should keep the terms consistent and avoid the confusion with description.

Reply:
@Jym77 I also haven't liked "description" from the start, so I'm up for changing it.

However, using "clearly identified location" would perhaps raise expectations of a programmatically determinable identifier. That's why I would prefer to retain "clearly labeled location" and change just "description" to something else. I've tried rewriting the definition using "identification" but I found it awkward. I'll try to come up with another alternative.